modules/tensor_mechanics/src/actions/GeneralizedPlaneStrainAction.C

```diff
@@ -41,12 +41,10 @@ GeneralizedPlaneStrainAction::act()
 {
   if (_current_task == "add_kernel")
   {
-    std::string k_type("GeneralizedPlaneStrainOffDiag");
+    std::string k_type = "GeneralizedPlaneStrainOffDiag";
     InputParameters params = _factory.getValidParams(k_type);
-
     params.set<std::vector<NonlinearVariableName> >("displacements") = _displacements;
-
-    params.set<std::vector<VariableName> >("scalar_strain_zz") = {_scalar_strain_zz};
+    params.set<std::vector<VariableName> >("scalar_strain_zz") = { _scalar_strain_zz };
```

Comment:
It won't be the first time. In fact it's probably at least the 3rd time since MOOSE's inception. There was David's MOOSE 2.0 rewrite that wasn't quite correct on ownership when everything got merged. He was trying to rewrite large parts of MOOSE while incorporating changes from Derek and I. After that, we didn't have any whitespace rules so I applied a patch to remove all whitespace messing up ownership there. I'm not sure how big of a deal it really is. I don't think we should really care about total line ownership. Some of us write a lot more code than others and that's not necessarily a good thing. In fact usually less experienced coders tend to repeat themselves or use longer verbose algorithms where standard template algorithms can be used instead.

I suppose somebody could write a script that creates a series of commits by owners based on current line ownership. There is a pragma that can be added to source code to tell clang-format to not touch lines of code in a source file. This script could run git-blame (we already have one that does that pretty well by the way) and insert this pragma all over among lines of codes to control which lines are formatted for a specific invocation. We could create a bunch of ownership preserving commits this way. I don't think I'm motivated enough to do that. 😄